test/UnitPerf.cpp

```diff
@@ -86,13 +94,29 @@ void UnitPerf::invokeWSDTest()
     std::cerr << "startup: " << _timer->elapsedTime().count() << "us\n";
     _timer->restart();
 
-    testPerf();
+    testPerf("writer", "odt", "/../traces/perf-writer.txt");
 
-    std::cerr << "test: " << _timer->elapsedTime().count() << "us\n";
+    testPerf("calc", "ods", "/../traces/perf-calc.txt");
+
+    long cpuTime = _timer->elapsedTime().count();
+
+    std::cerr << "test: " << cpuTime << "us\n";
 
     exitTest(TestResult::Ok);
 }
 
+//Called when document loading process starts e.g. setup finishes
+void UnitPerf::onDocumentLoading()
+{
+    stats->perfSetupLoadEditPhases("Setup");
```

Comment:
Hmm; sounds like stats->endPhase(foo) might be a better name (?) I assume that ends it (?)

Reply:
Yeah that would make sense, this gets called when the document is getting downloaded from storage (wopi host), so i assume from the beginning of the unit test to then is the setup phase?